golang/internal/helper/image/image.go

```diff
@@ -246,7 +247,11 @@ func checkRemote(ctx context.Context, check remoteCheck) (err error) {
 	craneOpts := []crane.Option{}
 
 	if check.encodedAuth != "" {
-		craneOpts = append(craneOpts, crane.WithAuth(authn.FromConfig(authn.AuthConfig{Auth: check.encodedAuth})))
+		basicAuth, convertError := authConfigToBasicAuth(check.encodedAuth)
+		if err != nil {
```

Comment:
I assume you wanted to check `convertError`.

Reply:
Oops, thanks.